glue_jupyter/link.py

```diff
@@ -10,12 +10,14 @@ def _is_echo(link):
 
 
 class link(object):
-    def __init__(self, source, target, f1=lambda x: x, f2=lambda x: x):
+    def __init__(self, source, target, f1=lambda x: x, f2=lambda x: x, forwards=True, backwards=True):
         self.source = source
         self.target = target
 
-        self._link(source, target, 'source', f1, True)
-        self._link(target, source, 'target', f2)
+        if forwards:
+            self._link(source, target, 'source', f1, True)
+        if backwards:
+            self._link(target, source, 'target', f2)
```

Comment:
we could also use dlink (which traitlets and ipywidgets has as well), to keep the api similar. If you prefer this i'm ok with it. I wonder about the last argument (sync_directly), when forwards is False, they will not be synced.

Reply:
Oh I see, I hadn't realized dlink does what we need already - will fix.